spec.html

```diff
@@ -6506,14 +6503,13 @@ <h1>CreateMutableBinding ( _N_, _D_ )</h1>
 
         <emu-clause id="sec-object-environment-records-createimmutablebinding-n-s">
           <h1>CreateImmutableBinding ( _N_, _S_ )</h1>
-          <p>The concrete Environment Record method CreateImmutableBinding is never used within this specification in association with object Environment Records.</p>
+          <p>The CreateImmutableBinding concrete method of an object Environment Record is never used within this specification.</p>
```

Comment:
Can we just delete it then?

Reply:
Not trivially, since we say it exists for every Environment Record type.

Also, that's not particularly related to this PR.